src/client/browser/service-worker.js

```diff
@@ -0,0 +1,47 @@
+const MAX_RETRY   = 10;
+const RETRY_DELAY = 500;
+
+const PAGE_FETCH_MODE = 'navigate';
+
+function delay (ms) {
+    return new Promise(r => setTimeout(r, ms));
+}
+
+async function tryGetResponse (request) {
+    try {
+        return { response: await fetch(request) };
```

Comment:
Check that `fetch` is not overridden.

Reply:
`fetch` cannot be overridden here, since we are on the idle page

Reply:
Ok